### PR TITLE
fix #1295

### LINKED ISSUE #1295: Javadoc for RuleChain contains errors
* The class modifier `static` is not necessary.
* The method calls are missing closing parentheses.

In addition, the documentation should make clear that the `RuleChain` cannot be used to define the order of existing rules, e.g. the following does not work as expected:

```
@Rule
public LoggingRule middle = new LoggingRule("middle rule");
@Rule
public RuleChain chain = RuleChain
    .outerRule(new LoggingRule("outer rule"))
    .around(middle)
    .around(new LoggingRule("inner rule"));
```

In this code, `middle` is executed twice.

### PR DESCRIPTION
**Pull request by Harrisnick**
Removed error from `RuleChain` Javadoc and clarified how it works with existing rules:

- Removed `static` modifier
- Added missing closing parenthesis of method calls
- Added clarification

### CODE REVIEW

File: src/main/java/org/junit/rules/RuleChain.java [hunk 1]
```diff
@@ -37,6 +37,22 @@
  * finished middle rule
  * finished outer rule
  * </pre>
+ * 
+ * {@code RuleChain} cannot be used to define the order of existing rules.</br>
```
Comment (marcphilipp) [inline src/main/java/org/junit/rules/RuleChain.java]: Please remove this `</br>` tag.

**Comment by Harrisnick** [comment]
</br> tag removed

File: src/main/java/org/junit/rules/RuleChain.java (hunk 1 above)
Comment (Harrisnick) [inline src/main/java/org/junit/rules/RuleChain.java]: </br> tag removed

File: src/main/java/org/junit/rules/RuleChain.java [hunk 2]
```diff
@@ -37,6 +37,22 @@
...
+ * &#064;Rule
+ * public LoggingRule middle = new LoggingRule("middle rule");
+ * 
+ * &#064;Rule
+ * public RuleChain chain = RuleChain
+ *                          .outerRule(new LoggingRule("outer rule"))
+ *                          .around(middle)
+ *                          .around(new LoggingRule("inner rule"));
+ * </pre>
+ * <p>
```
Comment (marcphilipp) [inline src/main/java/org/junit/rules/RuleChain.java]: This `<p>` tag should be at the beginning of the paragraph not its end, shouldn't it?

Comment (Harrisnick) [inline src/main/java/org/junit/rules/RuleChain.java]: I doubt that the `<p>` tag is required at all. It doesn't add anything to the layout really. I will remove it.

Comment (marcphilipp) [inline src/main/java/org/junit/rules/RuleChain.java]: It's usually a good idea to take a look at the generated Javadoc locally. You can generate it using Maven. Let me know if you need more detailed instructions.

Comment (Harrisnick) [inline src/main/java/org/junit/rules/RuleChain.java]: I assumed that the <p> tag was required to add some spacing at the end - this clearly wasn't the case. I used the Javadoc view in eclipse to check my changes. Now I also checked the Javadocs generated by maven and it looks alright.

**Comment by marcphilipp** [comment]
@Harrisnick Thanks! Please add a description of this improvement to the [draft of the release notes](https://github.com/junit-team/junit4/wiki/4.13-Release-Notes).

**Pull request closed without merging**